### PR TITLE
A few minor patches

### PR DESCRIPTION
**Pull request by swsch**
* two small fixes to README increase chance of running tests successfully
  * fix for locale senstive datetime string literal in query_test
  * fix table name in rpc_test

### CODE REVIEW

File: test/rpc_test.exs [hunk 1]
```diff
@@ -165,7 +165,7 @@ defmodule RPCTest do
       # %Tds.Parameter{direction: :input, name: "@8", type: :uuid, value: nil}]
     ] 
     assert [{1, nil}] = query(sql, params)
-    assert :ok = query("DROP TABLE dbo.TestTable", [])
+    assert :ok = query("DROP TABLE posts", [])
```
Comment (mobileoverlord) [inline test/rpc_test.exs]: I notice this wasn't working to drop either, however at this point its not the posts table

**Comment by mobileoverlord** [comment]
Thanks for the PR! :) I added 1 note where you modified it to drop the table that wasn't in use for that test.

File: test/rpc_test.exs (hunk 1 above)
Comment (swsch) [inline test/rpc_test.exs]: line 129 created table "posts" at the beginning of the test we're closing here, "TestTable" is used elsewhere.

Comment (mobileoverlord) [inline test/rpc_test.exs]: its been a long day, sorry bout that

Comment (swsch) [inline test/rpc_test.exs]: oh boy, do I know that feeling :-)

**Pull request merged**